bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -61,7 +61,7 @@
  */
 @Component(immediate = true)
 public class HttpUtil {
-    private static Logger logger = LoggerFactory.getLogger(HttpUtil.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(HttpUtil.class);
```

Comment:
actually we have `logger` all over the place.

Reply:
is this documented somewhere?

Reply:
@kaikreuzer and I used to agree that if we use a static logger then its a `private static final LOGGER` but that is maybe an openHAB rule :-)

Reply:
ah, good point. Its getting much too warm in here... I agree :-)